src/main/service.js

```diff
@@ -80,9 +80,9 @@ function logout(req, res, next) {
  *                 - if the password is incorrect
  */
 function changePassword(req, res, next) {
-    application.changePassword(process.pid, req.body.newPassword).then(
+    application.changePassword(req.body).then(
         businessUser => {
-            let token = jwt.sign({username: req.body.username}, process.env.SECRET_KEY, {
+            let token = jwt.sign({username: req.body.username, type: businessUser.getUserType()}, process.env.SECRET_KEY, {
                 expiresIn: "1h" // token expires in 15 minutes
```

Comment:
the returned business user is to update fields outside the token. you can use `req.body.callingUser` to get this information. 
What's important now is that the token's timer should be updated on almost *every* request to the server, otherwise it'll expire after 1 hour even if the user was busy all this time. Currently the timer is updated only at `login`/`changePassword`

Reply:
It is a logic that should be done both in the FE and the server.
this is for different PR.